Comp. Sci./SQL/What is SQL?/what-is-an-rdbms.md

```diff
@@ -0,0 +1,55 @@
+# What is an RDBMS?
+author: Nick Daminov
+
+levels:
+
+  - beginner
+
+  - basic
+
+type: normal
+
+category: must-know
+
+links:
+
+  - '[What is Relational Database?](http://searchsqlserver.techtarget.com/definition/relational-database)'
+  - '[What is RDBMS?](http://searchsqlserver.techtarget.com/definition/relational-database-management-system)'
+
+---
+## Content
+
+In order to learn **SQL** we must understand what a **RDBMS** is. **RDBMS** stands for *Relational Database Management System*, a *Database Management System* that allows the user to create, update and delete the *Relational Database*.
+
+*Relational Database* stores data in a form of well-defined tables from which we can request this data in various ways without changing the internal database organisation.
+
+To clear things out let's take a look at a few most widely spread software implementations of such RDBMS's:
+ - Oracle Database (by Oracle Corp.)
+ - MySQL (by Oracle Corp.)
+ - Microsoft SQL Server (by Microsoft)
+ - PostgreSQL (by PostgreSQL G.D.G)
+
+All of the above solutions are written in *C, C++ and assembly languages*. In short, **RDBMS** is a program that sits on the server and stores data (in a very efficient way) on a hard drive in some logical order.
+
+Then the user operates on these RDBMS's with *Query Languages*. The one most common and widely used *Query Language* is **SQL**.
+
+---
+## Practice
+
+What language are RDBMS's usually written in?
+???
+
+* C, C++ or assembly
+* SQL
+* MySQL
+* Java
+
+---
+## Revision
+
+What is RDBMS (Relational Database Management System)?
+???
+
+* It is a piece of software that sits on a server and operates the data using high level language.
+* It is a piece of software that operates the data using SQL or some other Query Language.
```

Comment:
for non-english speakers, these two answers are not distinct enough. diversify the incorrect answers more conceptually, instead of using the phrasing.